kokoro/config/test/ops_agent/presubmit/windows_x86_64_uap_plugin.gcl

```diff
@@ -0,0 +1,11 @@
+import 'common.gcl' as common
```

Comment:
Should we leave windows out for now since we are not building it?

Reply:
Done, removed.